doc/languages-frameworks/python.section.md

```diff
@@ -299,13 +304,14 @@ python3Packages.buildPythonApplication rec {
     hash  = "sha256-Pe229rT0aHwA98s+nTHQMEFKZPo/yw6sot8MivFDvAw=";
   };
 
-  nativeBuildInputs = with python3Packages; [
-    setuptools
+  build-system = [
+    python3Packages.setuptools
+    python3Packages.wheel
   ];
 
-  propagatedBuildInputs = with python3Packages; [
-    tornado
-    python-daemon
+  dependencies = [
+    python3Packages.tornado
+    python3Packages.python-daemon
   ];
```

Comment:
I don't think moving away from recommending `with python3Packages` is very ergonomic.

Reply:
Sure, not gonna argue over that.
It was easier to restore it to not use `with` during a rebase.